Podfile.lock

```diff
@@ -602,7 +602,7 @@ DEPENDENCIES:
   - SVProgressHUD (= 2.2.5)
   - WordPress-Editor-iOS (~> 1.19.8)
   - WordPressAuthenticator (>= 3.2.2, ~> 3.2)
-  - WordPressKit (~> 4.58.2-beta.1)
+  - WordPressKit (>= 4.58.1, ~> 4.58)
```

Comment:
On `trunk`, WordPressKit had been changed to `~> 4.58.2-beta.1`. But that's unnecessary to fetch that version based on the `>= 4.58.1, ~ 4.58` rule already in place. `bundle exec pod update WordPressKit` is enough, as demonstrated by the SHA1 for WordPressKit not having changed in this diff.